tasks/tasks.go

```diff
@@ -38,12 +40,37 @@ func LoadStorageEngine(ctx context.Context, config *tsbridge.Config) (storage.Ma
 	}
 }
 
-// Sync updates all configured metrics.
-func Sync(ctx context.Context, config *tsbridge.Config, metrics *tsbridge.Metrics, store storage.Manager) error {
-	metricCfg, err := tsbridge.NewMetricConfig(ctx, config, store)
+// Checks if the metric config file has been updated, and updates the cached file stats if it has.
```

Comment:
https://golang.org/doc/effective_go.html#commentary

Comments must start with the name of the func. Also `metricConfigFileUpdated` is a mouthful. Perhaps `func isCfgUpdated` might be better, since it returns true if it is?

(or pick a more suitable name)

Reply:
I changed it to `isMetricCfgUpdated` since there is another struct called `tsbridge.Config`.